### PR TITLE
Fix url for util.api.search

### PR DESCRIPTION
**Pull request by pythe**
We saw that search didn't work because it was running _queryObjToString on a string.

### CODE REVIEW

**Comment by jdd1260** [comment]
@prattsj or @robert-wallis do you think that you could approve this pull request?

**Comment by prattsj** [comment]
Sure thing. Will take a look and merge tonight. Gosh, this old thing could use quite a few updates. Thanks for the submission!

File: tests/unit/run.js [hunk 1]
```diff
@@ -11,5 +11,6 @@ module.exports = {
     "test-resource-projectmembership": require('./test-resource-projectmembership.js'),
     "test-resource-story": require('./test-resource-story.js'),
     "test-resource-task": require('./test-resource-task.js'),
+    "test-resource-utils": require('./test-utils.js'),
```
Comment (prattsj) [inline tests/unit/run.js]: Since utils isn't a resource, can we just call this script `"test-utils"`?

**Comment by prattsj** [comment]
Very minor nitpick about an npm script name, but otherwise LGTM. This module will probably get a fairly significant rewrite with more up-to-date standards, so I'm overlooking anything related to style for now. Thanks again for the submission!

**Pull request merged**